File: AssetManager/UserInterface/CustomControls/CorrectedTreeView.vb
Public Class CorrectedTreeView
    Inherits TreeView

    Protected Overrides Sub WndProc(ByRef m As Message)
        ' Suppress WM_LBUTTONDBLCLK
        ' Fixes bug that has existed since Vista...  (╯°□°)╯︵ ┻━┻
        If m.Msg = &H203 Then
            m.Result = IntPtr.Zero
        Else
            MyBase.WndProc(m)
        End If
    End Sub

End Class

File: AssetManager/UserInterface/CustomControls/CorrectedTreeView.cs
using System;
using System.Windows.Forms;
namespace AssetManager
{
    public class CorrectedTreeView : TreeView
    {

        protected override void WndProc(ref Message m)
        {
            // Suppress WM_LBUTTONDBLCLK
            // Fixes bug that has existed since Vista...  (╯°□°)╯︵ ┻━┻
            if (m.Msg == 0x203)
            {
                m.Result = IntPtr.Zero;
            }
            else
            {
                base.WndProc(ref m);
            }
        }

    }
}
